src/taipy/core/config/data_node_config.py

```diff
@@ -157,49 +163,72 @@ class DataNodeConfig(Section):
             _REQUIRED_READ_FUNCTION_GENERIC_PROPERTY,
             _REQUIRED_WRITE_FUNCTION_GENERIC_PROPERTY,
         ],
-        _STORAGE_TYPE_VALUE_JSON: [_REQUIRED_DEFAULT_PATH_JSON_PROPERTY],
+        _STORAGE_TYPE_VALUE_JSON: [],
         _STORAGE_TYPE_VALUE_PARQUET: [],
     }
 
     _OPTIONAL_PROPERTIES = {
```

Comment:
The `_OPTIONAL_PROPERTIES` was not used anywhere before.

I updated it to a dictionary, mapping optional property keys to the default value (for some attributes, most are None).